test/testBrowserLanguage.js

```diff
@@ -0,0 +1,112 @@
+import browserLanguage from '../src/nginxconfig/util/browser_language';
+
+class mockLocales{
+    static languages = [];
+
+    static language = null;
+    IntlBackup = null;
+
+    setNavigatorLanguages(langs){
+        mockLocales.languages = langs;
+        return this;
+    }
+
+    setNavigatorLanguage(lang){
+        mockLocales.language = lang;
+        return this;
+    }
+
+    setDateTimeLocale(locale) {
+        const newDateTimeForMat = new Intl.DateTimeFormat(locale);
+        this.IntlBackup = Intl;
+        if (!locale){
+            // eslint-disable-next-line no-global-assign
+            Intl = undefined;
+            return;
+        }
+
+        // eslint-disable-next-line no-global-assign
+        Intl = {
+            DateTimeFormat(){
+                return newDateTimeForMat;
+            },
+        };
+        return this;
+    }
+
+    restoreDateTimeLocale(){
+        // eslint-disable-next-line no-global-assign
+        Intl = this.IntlBackup;
+        return this;
+    }
+}
+Object.defineProperty(window.navigator,'languages',{get:() => {
+    return mockLocales.languages ?? [];
+}});
+
+Object.defineProperty(window.navigator,'language',{get:() => {
+    return mockLocales.language ?? null;
+}});
+
+
+describe('#testGetBrowserLanguage', ()=> {
```

Comment:
```suggestion
describe('#browserLanguage', ()=> {
```
Let's describe the actual method we're testing